e2e/commands/start/start_test.go

```diff
@@ -434,6 +434,21 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 		})
 	})
 
+	When("--enable-owasp-dependency-check is passed", func() {
+		owaspDependencyCheck := "true"
+
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath:                projectPath,
+				testutil.StartFlagEnableOwaspDependencyCheck: owaspDependencyCheck,
+			}
+		})
+
+		It("Checks if the enable owasp dependecy was set with true", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`\"enable_owasp_dependency_check\": %s,\n`, owaspDependencyCheck)))
```

Comment:
Can we add a new Expected that check that the log message "Running OwaspDependencyCheck" is printed? With this we can sure that dependency check is started.

Reply:
The assert added is confusing. If the test is about checking that dependency check is running why `Expect(session.Out.Contents()).To(ContainSubstring("{HORUSEC_CLI} The tool was ignored for run in this analysis: OwaspDependencyCheck"))` was added instead `Expect(session.Out.Contents()).To(ContainSubstring("Running OwaspDependencyCheck"))`?